src/helpers/config_access.js

```diff
@@ -64,6 +64,11 @@ class ConfigAccess {
     if( oServerConfig.infinityRestServerUrl.endsWith('/') ) {
       oServerConfig.infinityRestServerUrl = oServerConfig.infinityRestServerUrl.slice(0, -1)
     }
+
+    // Specify our own internal list of well known portals to exclude (if one not specified)
+    if( !oServerConfig.excludePortals ) {
+      oServerConfig.excludePortals = ["pxExpress", "Developer", "pxPredictionStudio", "pxAdminStudio", "pyCaseWorker", "pyCaseManager7"];
```

Comment:
I don't think we should inject this kind of "buried" logic. It could lead to too much confusion. Instead, maybe write out a console warning and indicate what we think it should be set to.

Reply:
And, perhaps, we could better trap the portal failure condition and render something on the page other than the blank screen that appears now.

Reply:
We allow overriding the list, the above line is only if that entry was totally omitted from sdk config.  If SDK does support one of these, the line can be modified here (and within the default sdk_config.json.

Reply:
> And, perhaps, we could better trap the portal failure condition and render something on the page other than the blank screen that appears now.

What's worse is SDK not working with a freshly generated Constellation app.  With these changes, the admin/author user would experience the WebPortal and most functionality appears to work just fine.  Admin/Author can create their own custom portal without supported entries like Pulse and set that to default for the AG if they so desire.  Most will be fine with users going to the custom trimmed down portal and admins going to the full IMHO.

Reply:
Added console log indicating that the excludePortals setting was not found and hence using this default.